data/operators/highway/toll_gantry.json

```diff
@@ -0,0 +1,132 @@
+{
+  "properties": {
+    "path": "operators/highway/toll_gantry",
+    "preserveTags": ["^name"],
```

Comment:
Is `preserveTags` needed if none of the entries specifies a `name`? In most if not all of these systems, each individual gantry has a unique name that shows up on maps and bills.

Reply:
> Is `preserveTags` needed if none of the entries specifies a name?
It is not needed, but also no harm in having it there.